make: *** [GNUmakefile:24: update] Error 1

<!--
Thank you for reporting an issue.

Please fill in as much of the template below as you're able.

Version:      if you know it, otherwise use the git revision
Installation: How did you install wallabag? Using git clone, the docker image, an installer, downloading the package, etc.
PHP version:  The version of PHP you are using
OS:           The host running wallabag
Database:     The storage system your instance is using (SQLite, MySQL/MariaDB or PostgreSQL) with the version
Parameters:   Paste the content of your app/config/parameters.yml (hide sensitive stuff if you want)
-->
### Environment

* **Version**: d8ef4ae6a
* **Installation**: [on-a-dedicated-web-server](https://doc.wallabag.org/en/admin/installation/installation.html#on-a-dedicated-web-server-recommended-way)
* **PHP version**: 8.2
* **OS**: Debian 12
* **Database**: mysql
* **Parameters**:

<details>
  <summary>My <code>app/config/parameters.yml</code> is:</summary>

  ```
# This file is auto-generated during the composer install
parameters:
    database_driver: pdo_mysql
    database_host: 127.0.0.1
    database_port: null
    database_name: wallabag
    database_user: wallabag
    database_password:
    database_path: null
    database_table_prefix: wallabag_
    database_socket: null
    database_charset: utf8mb4
    domain_name: 'https://wallabag.rollenspiel.monster'
    server_name: 'RollenspielMonster Wallabag'
    locale: de
    secret: 
    twofactor_auth: true
    twofactor_sender: noreply@rollenspiel.monster
    fosuser_registration: true
    fosuser_confirmation: true
    fos_oauth_server_access_token_lifetime: 3600
    fos_oauth_server_refresh_token_lifetime: 1209600
    from_email: noreply@rollenspiel.monster
    rss_limit: 50
    rabbitmq_host: localhost
    rabbitmq_port: 5672
    rabbitmq_user: guest
    rabbitmq_password: guest
    rabbitmq_prefetch_count: 10
    redis_scheme: tcp
    redis_host: localhost
    redis_port: 6379
    redis_path: null
    redis_password: null
    sentry_dsn: null
    mailer_dsn: 'smtp://mail.rollenspiel.network'
  ```
</details>

### What steps will reproduce the bug?

<!--
Enter details about your bug and how to reproduce it
-->

update the instance with `make update`

```
composer.phar not found, we'll see if composer is installed globally.
HEAD is now at d8ef4ae6a Merge pull request #6646 from weblate/weblate-wallabag-messages
Installing dependencies from lock file
Verifying lock file contents can be installed on current platform.
Nothing to install, update or remove
Package guzzlehttp/ringphp is abandoned, you should avoid using it. No replacement was suggested.
Package guzzlehttp/streams is abandoned, you should avoid using it. No replacement was suggested.
Package hoa/compiler is abandoned, you should avoid using it. No replacement was suggested.
Package hoa/consistency is abandoned, you should avoid using it. No replacement was suggested.
Package hoa/event is abandoned, you should avoid using it. No replacement was suggested.
Package hoa/exception is abandoned, you should avoid using it. No replacement was suggested.
Package hoa/file is abandoned, you should avoid using it. No replacement was suggested.
Package hoa/iterator is abandoned, you should avoid using it. No replacement was suggested.
Package hoa/math is abandoned, you should avoid using it. No replacement was suggested.
Package hoa/protocol is abandoned, you should avoid using it. No replacement was suggested.
Package hoa/regex is abandoned, you should avoid using it. No replacement was suggested.
Package hoa/ruler is abandoned, you should avoid using it. No replacement was suggested.
Package hoa/stream is abandoned, you should avoid using it. No replacement was suggested.
Package hoa/ustring is abandoned, you should avoid using it. No replacement was suggested.
Package hoa/visitor is abandoned, you should avoid using it. No replacement was suggested.
Package hoa/zformat is abandoned, you should avoid using it. No replacement was suggested.
Package php-http/message-factory is abandoned, you should avoid using it. Use psr/http-factory instead.
Package sensio/framework-extra-bundle is abandoned, you should avoid using it. Use Symfony instead.
Package true/punycode is abandoned, you should avoid using it. No replacement was suggested.
Package php-cs-fixer/diff is abandoned, you should avoid using it. No replacement was suggested.
Generating optimized autoload files
Class SimpleHtmlDom\simple_html_dom_node located in ./vendor/mgargano/simplehtmldom/src/simple_html_dom.php does not comply with psr-0 autoloading standard. Skipping.
Class SimpleHtmlDom\simple_html_dom located in ./vendor/mgargano/simplehtmldom/src/simple_html_dom.php does not comply with psr-0 autoloading standard. Skipping.
54 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
> Incenteev\ParameterHandler\ScriptHandler::buildParameters
Updating the "app/config/parameters.yml" file
> bin/console cache:clear --no-warmup

In DefinitionErrorExceptionPass.php line 54:
                                                                                                                                       
  Class "Doctrine\Bundle\FixturesBundle\Fixture" not found while loading "Wallabag\AnnotationBundle\DataFixtures\AnnotationFixtures".  
                                                                                                                                       

Script bin/console cache:clear --no-warmup handling the post-cmd event returned with error code 1
Script @post-cmd was called via post-install-cmd
make: *** [GNUmakefile:24: update] Error 1
```

--- a/app/config/services.yml
+++ b/app/config/services.yml
@@ -33,7 +33,7 @@ services:
 
     Wallabag\AnnotationBundle\:
         resource: '../../src/Wallabag/AnnotationBundle/*'
-        exclude: '../../src/Wallabag/AnnotationBundle/{Controller,Entity}'
+        exclude: '../../src/Wallabag/AnnotationBundle/{Controller,Entity,DataFixtures}'
 
     Wallabag\ApiBundle\:
         resource: '../../src/Wallabag/ApiBundle/*'
@@ -41,7 +41,7 @@ services:
 
     Wallabag\CoreBundle\:
         resource: '../../src/Wallabag/CoreBundle/*'
-        exclude: ['../../src/Wallabag/CoreBundle/{Controller,Entity}', '../../src/Wallabag/CoreBundle/Event/*Event.php']
+        exclude: ['../../src/Wallabag/CoreBundle/{Controller,Entity,DataFixtures}', '../../src/Wallabag/CoreBundle/Event/*Event.php']
 
     # controllers are imported separately to make sure services can be injected
     # as action arguments even if you don't extend any base controller class
@@ -122,7 +122,7 @@ services:
 
     Wallabag\UserBundle\:
         resource: '../../src/Wallabag/UserBundle/*'
-        exclude: '../../src/Wallabag/UserBundle/{Controller,Entity}'
+        exclude: '../../src/Wallabag/UserBundle/{Controller,Entity,DataFixtures}'
 
     Doctrine\DBAL\Connection:
         alias: doctrine.dbal.default_connection

